recipe/meta.yaml

```diff
@@ -43,8 +43,10 @@ outputs:
         - arm-variant * {{ arm_variant_type }}  # [aarch64]
       host:
         - cuda-version {{ cuda_version }}
-        - expat                                 # [aarch64]
+        - expat                                 # [aarch64 or ppc64le]
         - gmp
+        - mpfr                                  # [ppc64le]
+        - ncurses                               # [ppc64le]
```

Comment:
Am seeing [this error on CI]( https://app.travis-ci.com/github/conda-forge/cuda-gdb-feedstock/jobs/604933810#L631 ):

```
  ERROR (cuda-gdb,bin/cuda-gdb): $RPATH/libtinfo.so.5 not found in packages, sysroot(s) nor the missing_dso_whitelist.
```

As the current version of [`ncurses` in conda-forge is `6.4`]( https://github.com/conda-forge/ncurses-feedstock/blob/456de0478ea4ef42d63eea3ec72adc24dfce01d6/recipe/meta.yaml#L1 ), which contains the `libtinfo.so.6` library, think this is too new for what `cuda-gdb` is looking for as it is looking for `libtinfo.so.5`. This was [last included in `ncurses` version `5.9`]( https://abi-laboratory.pro/index.php?view=objects_report&l=ncurses&v1=5.9&v2=6.0 )